src/Backend/Core/Engine/Base/Action.php

```diff
@@ -96,15 +95,15 @@ public function getAction(): string
      */
     public function checkToken(): void
     {
-        $fromSession = (\SpoonSession::exists('csrf_token')) ? \SpoonSession::get('csrf_token') : '';
-        $fromGet = $this->getContainer()->get('request')->query->get('token');
+        $fromSession = BackendModel::getSession()->get('csrf_token', '');
```

Comment:
Well, sessions should not be started if not needed. Because it makes it harder to cache, and generates useless cookies.